tasks/main-stable.yml

```diff
@@ -1,7 +1,7 @@
 ---
-  # Install packages before fetching dynamic URLs, since python-requests
-  # is required by the URL-fetching Ansible module.
-- include: packages.yml
+# Install packages before fetching dynamic URLs, since python-requests
+# is required by the URL-fetching Ansible module.
+- import_tasks: packages.yml
```

Comment:
All these `import_tasks` changes require Ansible v2.4... but we're already specifying that as the minimum Ansible version for the role, so :ok_hand:!